### DISCUSSION TITLE
Integrating Blocknote as default RTE

### DISCUSSION THREAD

**Discussion by reyren18** [GrapesJS/grapesjs#5927] [Q&A]
Hi! I was going through the grapesjs docs and was specifically looking for how to change the default RTE. It seems that the documentation is pretty inclined towards just CKeditor, however I wanted to integrate Blocknote (https://www.blocknotejs.org/) as the custom RTE. If someone has already done something similar with some other rte, any help would be appreciated.

**Comment by artf** [comment]
There is a guide on how to integrate your RTE https://grapesjs.com/docs/guides/Replace-Rich-Text-Editor.html
